ocpi-endpoints-common/src/main/scala/com/thenewmotion/ocpi/common/AuthorizationRejectionHandler.scala

```diff
@@ -0,0 +1,30 @@
+package com.thenewmotion.ocpi.common
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.server.directives.BasicDirectives
+import akka.http.scaladsl.server.directives.MiscDirectives
+import com.thenewmotion.ocpi.msgs
+
+// TODO: Could this move to OcpiRejectionHandler?
```

Comment:
yes, this makes sense

I also see that the toplevel route contains stuff that's mostly important for the registration part of OCPI. This should be moved to a separate module, maybe as part of TNM-3843